pkg/webhooks/machineset_webhook_test.go

```diff
@@ -326,98 +420,120 @@ func TestMachineSetCreation(t *testing.T) {
 }
 
 func TestMachineSetUpdate(t *testing.T) {
-	awsClusterID := "aws-cluster"
+	//awsClusterID := "aws-cluster"
 	awsRegion := "region"
-	defaultAWSProviderSpec := &machinev1.AWSMachineProviderConfig{
-		AMI: machinev1.AWSResourceReference{
-			ID: pointer.StringPtr("ami"),
-		},
-		InstanceType:      defaultAWSX86InstanceType,
-		UserDataSecret:    &corev1.LocalObjectReference{Name: defaultUserDataSecret},
-		CredentialsSecret: &corev1.LocalObjectReference{Name: defaultAWSCredentialsSecret},
-		Placement: machinev1.Placement{
-			Region: awsRegion,
-		},
-	}
```

Comment:
why we are removing this block and also see lot of code commented,

Reply:
Fixed it in the latest commit